cocos/audio/android/jni/cddandroidAndroidJavaEngine.h

```diff
@@ -53,6 +53,7 @@ namespace CocosDenshion {
             void resumeAllEffects();
             void stopEffect(unsigned int nSoundId);
             void stopAllEffects();
+            void loadEffect(const char* pszFilePath, char* loadEffectName);
```

Comment:
I think don't have to declare this function here.
It could be declared and implemented as a static function in `cddandroidAndroidJavaEngine.cpp`.